demo/src/app/components/timepicker/demos/seconds/timepicker-seconds.ts

```diff
@@ -9,6 +9,6 @@ export class NgbdTimepickerSeconds {
   seconds = true;
 
   toggleSeconds() {
-      this.seconds = !this.seconds;
+    this.seconds = !this.seconds;
```

Comment:
I think it's manual edit. I must have been bothered by the unusual indentation level.